src/vs/base/browser/ui/selectBox/selectBoxCustom.ts

```diff
@@ -397,8 +397,16 @@ export class SelectBoxList implements ISelectBoxDelegate, IDelegate<ISelectOptio
 		container.appendChild(this.selectDropDownContainer);
 
 		this.layoutSelectDropDown();
+
+		// {{SQL EDIT}}
```

Comment:
To match the rest this should be `// {{SQL CARBON EDIT}}`

Reply:
Yes, please make this `// {{SQL CARBON EDIT}}` so that it's easy to search for all our patches to VS Code.